src/components/Dropdown/Dropdown.tsx

```diff
@@ -73,9 +72,14 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     }
   }, [parentQuery, lastTypedOrSubmittedValue, updateFocusedItem, setLastTypedOrSubmittedValue]);
 
-  useRootClose(containerRef, () => {
-    toggleDropdown(false);
-  }, { disabled: !isActive });
+  useGlobalListener('click', e => {
+    if (isActive) {
```

Comment:
useRootClose also closes the Dropdown when somebody hits the ESC key. I played around with it a bit and it feels a bit weird for the SearchBar to close on ESC but it's also what google does so 🤷 I just wanted to point this out in case people have other ideas.

Reply:
OH I missed that you added it below (and also have it in your commit msg lol)